scripts/vim-patch.sh

```diff
@@ -64,17 +64,17 @@ assign_commit_details() {
   if [[ ${1} =~ [0-9]\.[0-9]\.[0-9]{3,4} ]]; then
     # Interpret parameter as version number.
     vim_version="${1}"
-    vim_commit="v${1//./-}"
+    vim_commit="v${1}"
     local strip_commit_line=true
-    vim_commit_url="https://github.com/vim/vim/commit/${vim_commit}"
+    vim_commit_url="https://github.com/vim/vim/tree/${vim_commit}"
```

Comment:
Not in my testing.  In this code, `$vim_commit` will be a tag ref like `v7.4.723`.  E.g. compare:

https://github.com/vim/vim/commit/v7.4.723

to 

https://github.com/vim/vim/tree/v7.4.723